docs-developer/upgrading-profiles.md

```diff
@@ -78,3 +78,5 @@ At all times, `processProfile` only has code that converts the latest version
 of the Gecko profile format into the latest version of the processed profile
 format. All compatibility code is hosted outside of process-profile.js, in
 the respective *-profile-versioning.js file.
```

Comment:
I feel like the `*` needs to be escaped somehow here (or maybe we beed to put this word between backticks)